packages/vaadin-virtual-list/test/unlimited-size.test.js

```diff
@@ -1,6 +1,6 @@
 import { expect } from '@esm-bundle/chai';
 import { fixtureSync, oneEvent } from '@vaadin/testing-helpers';
-import { Virtualizer } from '..';
+import { Virtualizer } from '../virtualizer';
```

Comment:
Let's add `.js` extension for consistency.

Reply:
Done